traits-stubs/setup.py

```diff
@@ -10,16 +10,58 @@
 
 import setuptools
 
+
+def get_long_description():
+    """ Read long description from README.rst. """
+    with open("README.rst", "r", encoding="utf-8") as readme:
+        return readme.read()
+
+
 if __name__ == "__main__":
     setuptools.setup(
         name="traits-stubs",
-        version="0.1.0",
-        description="type annotation integration stubs for traits",
-        install_requires=["mypy", "traits"],
-        packages=["traits-stubs",
-                  "traits_stubs_tests",
-                  "traits_stubs_tests.examples"],
-        package_data={
-            'traits-stubs': ['./*.pyi', './**/*.pyi'],
-        },
+        version="6.1.0",
+        url="https://github.com/enthought/traits",
+        author="Enthought",
+        author_email="info@enthought.com",
+        classifiers=[
+            c.strip()
+            for c in """
+            Development Status :: 4 - Beta
+            Intended Audience :: Developers
+            Intended Audience :: Science/Research
+            License :: OSI Approved :: BSD License
+            Operating System :: MacOS :: MacOS X
+            Operating System :: Microsoft :: Windows
+            Operating System :: POSIX :: Linux
+            Programming Language :: C
```

Comment:
The stub does not support C code so it should not be indexed for the programming language C, right?

Reply:
Yes, I'll remove this. 

The classifiers are _almost_ the same as for Traits, except that I went with a maturity level of "4" rather than "5", and I added the "Typing :: Typed" classifier.